test/unit/resources/pg_ident_conf_test.rb

```diff
@@ -0,0 +1,27 @@
+# encoding: utf-8
+# copyright: 2017
+# author: Aaron Lippold, lippold@gmail.com
+# author: Rony Xavier, rx294@nyu.edu
+# license: All rights reserved
+
+require 'helper'
+require 'inspec/resource'
+
+describe 'Inspec::Resources::PGIdentConf' do
```

Comment:
I'd like to see the tests account for a pg_ident file that has multiple entries to ensure the resource is parsing the file properly and handing back all the lines to FilterTable.

Just adding a second line and changing these must_match statements to match an array of contents would be sufficient for that.